DiscoveryServiceSpecification/V3.0_SSP-001.yaml

```diff
@@ -11,83 +11,62 @@ info:
   license: 
     name: CC BY 4.0
     url: https://creativecommons.org/licenses/by/4.0/
-  version: V3.1.0_SSP-001
-  x-profile-identifier: https://admin-shell.io/aas/API/3/1/DiscoveryServiceSpecification/SSP-001
+  version: V3.0.1_SSP-001
+  x-profile-identifier: https://admin-shell.io/aas/API/3/0/DiscoveryServiceSpecification/SSP-001
```

Comment:
This line is not present in V3.0.1: was it is V3.0

Reply:
V3.0.1 (current main branch) has it: https://github.com/admin-shell-io/aas-specs-api/blob/a6956e56975c867494ef1d9701444cd2082342a6/DiscoveryServiceSpecification/V3.0_SSP-001.yaml#L14